lib/Property/ICalendar/CalAddress.php

```diff
@@ -53,7 +53,11 @@ public function getNormalizedValue()
             return $input;
         }
         list($schema, $everythingElse) = explode(':', $input, 2);
+        $schema = strtolower($schema);
+        if($schema == "mailto") {
```

Comment:
should be  ===

Reply:
strtolower() returns a string value, so there is no real difference between == and === here.
But you are right. When using a language where "0xFF" == "255", one should always compare strings with the === operator.